src/components/ContactSection/ContactSection.js

```diff
@@ -1,14 +1,65 @@
 import React from 'react';
+import styled from 'styled-components';
 import { FormattedMessage } from 'gatsby-plugin-intl';
+import { LinkedinSquare } from '@styled-icons/boxicons-logos/LinkedinSquare';
+import { Envelope } from '@styled-icons/boxicons-regular/Envelope';
+import { Phone } from '@styled-icons/boxicons-regular/Phone';
 
-import SectionHeader from 'styles/elements/SectionHeader';
+import { colors, breakPoints } from 'styles/base';
 import SectionWrapper from 'styles/elements/SectionWrapper';
+import SectionHeader from 'styles/elements/SectionHeader';
```

Comment:
keeping react Components in `styles` folder is very confusing. Let's thing about your architecture. Try to be generic and don't repeat yourself, but not like in rush. Think if this component will be reused (it has no so complicated styles). 

If you need some generic styled components like wrappers, maybe it is worth to create folder wrappers.